pom.xml

```diff
@@ -253,18 +263,38 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
any particular reason this was re-added?

Reply:
It doesn't compile without it, try removin git